### ISSUE TITLE
`vote <specific bot>` doesn't seem to work

### ISSUE THREAD

**Issue by felipe19930** [chr233/ASFEnhance#60]
I have two bots: `triplex` and `singer`.

When I run this in ASF-Ui "Commands" page:

```
vote triplex 1245620,1849900,108600,648800,698670,1637320,1142710,1462040,1703340,1455840,1794680
```

ASFEnhance seems to vote using `singer` instead of `triplex`, judging by ASF's response:

![image](https://user-images.githubusercontent.com/35700659/209406523-82b65ae7-5acb-4453-8752-72a95329afaf.png)

- ArchiSteamFarm 5.4.0.3
- ASFEnhance 1.7.4.0

Thanks!

**Comment by woctezuma** [comment]
What happens with:

```bash
!vote ASF
```

**Comment by felipe19930** [comment]
![image](https://user-images.githubusercontent.com/35700659/209414909-f9a94205-d02f-4aac-bf94-1d420e8efdb4.png)

**Comment by chr233** [comment]
seems works fine? any problem?

**Comment by felipe19930** [comment]
Sure, when I run `vote asf` to vote for *all bots*, then it works.

However, when I run `vote <specific bot>` to vote for *a specific bot*, it doesn't... :thinking:

**Comment by chr233** [comment]
i see, i will check later

**Comment by chr233** [comment]
fixed in `1.7.4.1`

**Comment by felipe19930** [comment]
Works perfectly now, thanks a lot @chr233!
